sub-packages/bionemo-moco/environment/moco.yaml

```diff
@@ -0,0 +1,324 @@
+      - frozenlist==1.4.1
+      - fsspec==2024.9.0
+      - gitdb==4.0.11
+      - gitpython==3.1.43
+      - h11==0.14.0
+      - httpcore==1.0.6
+      - httpx==0.27.2
+      - identify==2.6.1
+      - iniconfig==2.0.0
+      - ipdb==0.13.11
+      - ipykernel==6.29.5
+      - ipython==8.28.0
+      - isoduration==20.11.0
+      - jaxtyping==0.2.34
+      - jedi==0.19.1
+      - jmespath==1.0.1
+      - joblib==1.4.2
+      - json5==0.9.25
+      - jsonpointer==3.0.0
+      - jsonschema==4.23.0
+      - jsonschema-specifications==2023.12.1
+      - jupyter-client==8.6.3
+      - jupyter-core==5.7.2
+      - jupyter-events==0.10.0
+      - jupyter-lsp==2.2.5
+      - jupyter-server==2.14.2
+      - jupyter-server-terminals==0.5.3
+      - jupyterlab==4.2.5
+      - jupyterlab-pygments==0.3.0
+      - jupyterlab-server==2.27.3
+      - markdown-it-py==3.0.0
+      - matplotlib-inline==0.1.7
+      - mdurl==0.1.2
+      - mistune==3.0.2
+      - multidict==6.1.0
+      - mypy-extensions==1.0.0
+      - nbclient==0.10.0
+      - nbconvert==7.16.4
+      - nbformat==5.10.4
+      - nbval==0.11.0
+      - nest-asyncio==1.6.0
+      - nltk==3.9.1
+      - nodeenv==1.9.1
+      - notebook-shim==0.2.4
+      - numpy==1.26.4
+      - onnx==1.17.0
+      - overrides==7.7.0
+      - packaging==24.1
+      - pandocfilters==1.5.1
+      - parso==0.8.4
+      - pathspec==0.12.1
+      - pexpect==4.9.0
+      - platformdirs==4.3.6
+      - pluggy==1.5.0
+      - pot==0.9.5
+      - pre-commit==3.4.0
+      - prometheus-client==0.21.0
+      - prompt-toolkit==3.0.48
+      - protobuf==5.28.2
+      - psutil==6.0.0
+      - ptyprocess==0.7.0
+      - pure-eval==0.2.3
+      - pyasn1==0.6.1
+      - pydot==3.0.2
+      - pygments==2.18.0
+      - pyparsing==3.1.4
+      - pytest==8.3.3
+      - pytest-cov==4.1.0
+      - pytest-dependency==0.5.1
+      - pytest-timeout==2.2.0
+      - python-dateutil==2.9.0.post0
+      - python-json-logger==2.0.7
+      - pyzmq==26.2.0
+      - referencing==0.35.1
+      - regex==2024.9.11
+      - requests-mock==1.11.0
+      - rfc3339-validator==0.1.4
+      - rfc3986-validator==0.1.1
+      - rich==13.9.2
+      - rpds-py==0.20.0
+      - rsa==4.7.2
+      - ruff==0.0.292
+      - s3transfer==0.10.2
+      - send2trash==1.8.3
+      - six==1.16.0
+      - smmap==5.0.1
+      - sniffio==1.3.1
+      - soupsieve==2.6
+      - stack-data==0.6.3
+      - tach==0.13.0
+      - tenacity==8.5.0
+      - terminado==0.18.1
+      - testbook==0.4.2
+      - tinycss2==1.3.0
+      - tomli==2.0.2
+      - tomli-w==1.0.0
+      - tornado==6.4.1
+      - tqdm==4.66.5
+      - traitlets==5.14.3
+      - typeguard==2.13.3
+      - types-python-dateutil==2.9.0.20241003
+      - uri-template==1.3.0
+      - virtualenv==20.26.3
+      - wcwidth==0.2.13
+      - webcolors==24.8.0
+      - webencodings==0.5.1
+      - websocket-client==1.8.0
+      - werkzeug==3.0.4
+      - yarl==1.13.1
+prefix: /home/dreidenbach/mambaforge/envs/moco
```

Comment:
Is this portable? Seems like something about your env crept in?

Reply:
Good catch thank you. I updated the conda environment and also added a setup.sh to do everything from the start.